package/src/components/ConfirmDialog/ConfirmDialog.md

```diff
@@ -0,0 +1,19 @@
+### Overview
+
```

Comment:
Would it be useful to have dev docs here like this about where to find the corresponding MUI docs?

Example for this component:

`The ConfirmDialog component uses [Dialog](https://material-ui.com/api/dialog), DialogActions, DialogContent, DialogContentText, DialogTitle and Button from Material-UI.`